src/show/components/Navbar/view.jsx

```diff
@@ -8,31 +8,29 @@ import { MdOutlineExplore } from 'react-icons/md';
 import { CgArrowsExpandRight } from 'react-icons/cg';
 import { BsArrowBarLeft, BsFillPersonLinesFill, BsPersonCircle } from 'react-icons/bs';
 
-import { mock } from 'process/helpers';
 import Paths from 'process/routes/paths';
+import { mock, onFieldChange } from 'process/helpers';
 
 import { Button, Modal, TextField } from '../';
 
-const Navbar = ({ currentUser, signIn, signOut, signUp }) => {
+const Navbar = ({ currentUser, signIn, signOut, signUp, transient, updateProps }) => {
   const navigate = useNavigate();
 
   const [openLogInModal, setOpenLogInModal] = useState(false);
   const [openLogOutModal, setOpenLogOutModal] = useState(false);
   const [openSignUpModal, setOpenSignUpModal] = useState(false);
 
-  const [payload, setPayload] = useState({});
-
-  const onFieldChange = (keyName, value) => setPayload({ ...payload, [keyName]: value });
+  const dipatchUpdateProps = payload => updateProps(payload);
```

Comment:
You don't need this, just Paul needed it since he had do dispatch before passing updateProps. Delete this line and pass `updateProps` instead of `dipatchUpdateProps`.

Reply:
done